packages/datepicker/src/lib/Datepicker.stories.tsx

```diff
@@ -42,6 +42,81 @@ export const _Datepicker: StoryObj<typeof Datepicker> = {
   },
 };
 
+type DateRangePickerProps = {
+  startDate: Dayjs | null;
+  endDate: Dayjs | null;
+  onStartDateChange: (date: Dayjs | null) => void;
+  onEndDateChange: (date: Dayjs | null) => void;
+  startFieldProps?: {
+    label?: string;
+    helperText?: string;
+    helpTopicId?: string;
+    fullWidth?: boolean;
+  };
+  endFieldProps?: {
+    label?: string;
+    helperText?: string;
+    helpTopicId?: string;
+    fullWidth?: boolean;
+  };
+};
+
+export const DateRangePicker = ({
```

Comment:
can we move this into the story instead of defined outside? rn the code example isn't informative.